rook/toolbox-deploy.yaml

```diff
@@ -17,10 +18,15 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
         - name: rook-ceph-tools
-          image: rook/ceph:v1.5.3
-          command: ["/tini"]
-          args: ["-g", "--", "/usr/local/bin/toolbox.sh"]
+          image: rook/ceph:master
```

Comment:
Isn't `master` basically same as `latest`? I wouldn't like that

Reply:
I was as surprised as you are, but seems like Rook really insists on `master`:
- https://github.com/rook/rook/blob/master/deploy/examples/toolbox.yaml#L21
- https://hub.docker.com/r/rook/ceph/tags?page=1&name=latest
- https://hub.docker.com/r/rook/ceph/tags?page=1&name=master